src/App.jsx

```diff
@@ -30,34 +30,41 @@ function App() {
         Authorization: `Bearer ${import.meta.env.VITE_AIRTABLE_API_TOKEN}`
       }
     };
-    const url = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/${import.meta.env.VITE_TABLE_NAME}`;
+    const url = `https://api.airtable.com/v0/${import.meta.env.VITE_AIRTABLE_BASE_ID}/Final?view=Grid%20View&sort[0][field]=Title&sort[0][direction]=asc`;
 
     try {
       const response = await fetch(url, options);
       if (!response.ok) {
         throw new Error(`Error: ${response.status}`);
       } 
       const data = await response.json();
-      const todos = data.records.map(record => ({
-        title: record.fields.title,
+
+      const sortedRecords = data.records.sort((objectA, objectB) => {
+        const titleA = objectA.fields.Title.toLowerCase();
+        const titleB = objectB.fields.Title.toLowerCase();
+        if (titleA > titleB) return -1;
+        if (titleA < titleB) return 1;
+        return 0;
+    });
+
+      const todos = sortedRecords.map(record => ({
+        title: record.fields.Title,
         id: record.id
       }));
+      
       setTodoList(todos);
       setIsLoading(false);
     } catch (error) {
       console.error('Error fetching data:', error.message);
     }
   }
 
-  useEffect(() => {
-    fetchData();
-  }, []);
-
   useEffect(() => {
     getAsyncList().then(result => {
```

Comment:
You're setting the todoList from storage, then calling fetchData() which will overwrite the todoList. You can omit the local storage load,

Reply:
Also omitted the local storage load